From 921c5308e64a20eb3fb1c0e999ecf7b960bcfed1 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Mon, 23 Jul 2012 10:40:55 +0800
Subject: ENGR00218013-6 SDMA:Add HDMI script in SDMA firmware

Add support for hdmi-sdma script content in sdma firmware.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 firmware/imx/sdma/sdma-imx6q-to1.bin.ihex |  194 ++++++++++++++++-------------
 1 file changed, 105 insertions(+), 89 deletions(-)

diff --git a/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex b/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex
index b969928..2e561f0 100644
--- a/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex
+++ b/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex
@@ -1,5 +1,5 @@
 :1000000053444D4101000000010000001C000000AD
-:1000100025000000B000000078050000820200000A
+:1000100026000000B40000007A0600008202000002
 :10002000FFFFFFFF00000000FFFFFFFFFFFFFFFFDC
 :10003000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFD0
 :10004000FFFFFFFFFFFFFFFF6A1A0000FFFFFFFF38
@@ -9,92 +9,108 @@
 :10008000FFFFFFFFFFFFFFFF4C0400006E040000B6
 :10009000FFFFFFFF00180000FFFFFFFFFFFFFFFF54
 :1000A000000000000018000062180000161A00008E
-:1000B000E3C1DB57E35FE357F352016A8F00D500DA
-:1000C000017D8D00A005EB5D7804037D79042C7D16
-:1000D000367C79041F7CEE56000F6006057D0965AD
-:1000E000437E0A62417E20980A623E7E09653C7E1C
-:1000F00012051205AD026007037DFB55D36D2B98E9
-:10010000FB55041DD36DC86A2F7F011F03200048D3
-:10011000E47C5398FB55D76D150005780962C86AD1
-:100120000962C86AD76D5298FB55D76D1500150046
-:1001300005780A62C86A0A62C86AD76D5298FB5588
-:10014000D76D15001500150005780B62C86A0B62A3
-:10015000C86AD76D097CDF6D077F0000EB55004D45
-:10016000077DFAC1E35706980700CC680C6813C2F4
-:100170000AC20398D9C1E3C1DB57E35FE357F352E7
-:10018000216A8F00D500017D8D00A005EB5DFB5637
-:100190007804037D79042A7D317C7904207C700BFE
-:1001A0001103EB53000F6003057D0965377E0A627A
-:1001B000357E86980A62327E0965307E1205120508
-:1001C000AD026007027C065A8E98265A277F011FCF
-:1001D00003200048E87C700B11031353AF981500FF
-:1001E00004780962065A0962265AAE98150015006D
-:1001F00004780A62065A0A62265AAE98150015005B
-:10020000150004780B62065A0B62265A077C000020
-:10021000EB55004D067DFAC1E357699807000C685D
-:1002200013C20AC26698700B110313536C07017C4A
-:10023000D9C1FB5E8A066B07017CD9C1F35EDB592D
-:10024000D3588F0110010F398B003CC12B7DC05A50
-:10025000C85B4EC1277C88038906E35CFF0D11054E
-:10026000FF1DBC053E07004D187D700811007E077C
-:10027000097D7D07027D2852E698F852DB54BC02C6
-:10028000CC02097C7C07027D2852EF98F852D354A7
-:10029000BC02CC02097D0004DD988B00C052C8531B
-:1002A00059C1D67D0002CD98FF08BF007F07157D9C
-:1002B0008804D500017D8D00A005EB5D8F02120240
-:1002C0001202FF3ADA05027C3E071899A402DD0209
-:1002D000027D3E0718995E071899EB559805EB5D6E
-:1002E000F352FB546A07267D6C07017D55996B0715
-:1002F000577C6907047D6807027D010E2F9993588A
-:10030000D600017D8E009355A005935DA00602786E
-:100310000255045D1D7C004E087C6907037D025573
-:10032000177E3C99045D147F890693500048017D37
-:100330002799A099150006780255045D4F070255CC
-:10034000245D2F07017CA09917006F07017C012015
-:1003500093559D000700A7D9F598D36C6907047DD4
-:100360006807027D010E64999358D600017D8E00C6
-:100370009355A005935DA00602780255C86D0F7CC9
-:10038000004E087C6907037D0255097E7199C86D8E
-:10039000067F890693500048017D5C99A0999A993F
-:1003A000C36A6907047D6807027D010E8799935827
-:1003B000D600017D8E009355A005935DA0060278BE
-:1003C000C865045D0F7C004E087C6907037DC86525
-:1003D000097E9499045D067F890693500048017D4B
-:1003E0007F99A09993559D000700FF6CA7D9F598B8
-:1003F0000000E354EB55004D017CF598DD98E35483
-:10040000EB55FF0A1102FF1A7F07027CA005B49981
-:100410009D008C05BA05A0051002BA04AD04540471
-:100420000600E3C1DB57FB52C36AF352056A8F0033
-:10043000D500017D8D00A005EB5D7804037D790476
-:100440002B7D1E7C7904337CEE56000FFB55600734
-:10045000027DC36DD599041DC36DC8623B7E6006E5
-:10046000027D10021202096A357F1202096A327F88
-:100470001202096A2F7F011F03200048E77C099AB6
-:10048000FB55C76D1500150015000578C8620B6A8D
-:10049000C8620B6AC76D089AFB55C76D1500150039
-:1004A0000578C8620A6AC8620A6AC76D089AFB556D
-:1004B000C76D15000578C862096AC862096AC76D08
-:1004C000097C286A077F0000EB55004D057DFAC1C5
-:1004D000DB57BF9977C254040AC2BA99D9C1E3C1A4
-:1004E000DB57F352056A8F00D500017D8D00A00512
-:1004F000FB567804037D7904297D1F7C79042E7CCA
-:10050000E35D700D1105ED55000F6007027D065289
-:10051000329A2652337E6005027D10021202096A69
-:100520002D7F1202096A2A7F1202096A277F011FA2
-:1005300003200048EA7CE3555D9A1500150015007C
-:10054000047806520B6A26520B6A5C9A1500150055
-:10055000047806520A6A26520A6A5C9A15000478E0
-:100560000652096A2652096A097C286A077F000038
-:10057000DB57004D057DFAC1DB571B9A77C2540447
-:100580000AC2189AE3C1DB57F352056AFB568E0282
-:10059000941AC36AC8626902247D941EC36ED36E26
-:1005A000C8624802C86A9426981EC36ED36EC86299
-:1005B0004C02C86A9826C36E981EC36EC8629826FD
-:1005C000C36E6002097CC8626E02247D096A1E7FC8
-:1005D0000125004D257D849A286A187F04627AC21D
-:1005E000B89AE36E8F00D805017D8D00A005C86222
-:1005F0006E02107D096A0A7F0120F97C286A067F55
-:100600000000004D0D7DFAC1DB576E9A07000462B1
-:100610000C6AB59A286AFA7F04627AC258045404B4
-:08062000286AF47F0AC26B9AFC
+:1000B000061B0000E3C1DB57E35FE357F352016A1D
+:1000C0008F00D500017D8D00A005EB5D7804037DD8
+:1000D00079042C7D367C79041F7CEE56000F600677
+:1000E000057D0965437E0A62417E20980A623E7E54
+:1000F00009653C7E12051205AD026007037DFB55C4
+:10010000D36D2B98FB55041DD36DC86A2F7F011F3B
+:1001100003200048E47C5398FB55D76D1500057803
+:100120000962C86A0962C86AD76D5298FB55D76DD3
+:100130001500150005780A62C86A0A62C86AD76D98
+:100140005298FB55D76D15001500150005780B6208
+:10015000C86A0B62C86AD76D097CDF6D077F000033
+:10016000EB55004D077DFAC1E35706980700CC68B0
+:100170000C6813C20AC20398D9C1E3C1DB57E35F1D
+:10018000E357F352216A8F00D500017D8D00A00551
+:10019000EB5DFB567804037D79042A7D317C79047C
+:1001A000207C700B1103EB53000F6003057D096584
+:1001B000377E0A62357E86980A62327E0965307E15
+:1001C00012051205AD026007027C065A8E98265A67
+:1001D000277F011F03200048E87C700B1103135395
+:1001E000AF98150004780962065A0962265AAE983B
+:1001F0001500150004780A62065A0A62265AAE985B
+:1002000015001500150004780B62065A0B62265A79
+:10021000077C0000EB55004D067DFAC1E357699855
+:1002200007000C6813C20AC26698700B11031353BF
+:100230006C07017CD9C1FB5E8A066B07017CD9C1C2
+:10024000F35EDB59D3588F0110010F398B003CC18D
+:100250002B7DC05AC85B4EC1277C88038906E35CAE
+:10026000FF0D1105FF1DBC053E07004D187D7008F0
+:1002700011007E07097D7D07027D2852E698F8521D
+:10028000DB54BC02CC02097C7C07027D2852EF982B
+:10029000F852D354BC02CC02097D0004DD988B00D7
+:1002A000C052C85359C1D67D0002CD98FF08BF0087
+:1002B0007F07157D8804D500017D8D00A005EB5DCD
+:1002C0008F0212021202FF3ADA05027C3E071899E9
+:1002D000A402DD02027D3E0718995E071899EB55CE
+:1002E0009805EB5DF352FB546A07267D6C07017D90
+:1002F00055996B07577C6907047D6807027D010EDD
+:100300002F999358D600017D8E009355A005935DDB
+:10031000A00602780255045D1D7C004E087C69072A
+:10032000037D0255177E3C99045D147F8906935026
+:100330000048017D2799A099150006780255045DB3
+:100340004F070255245D2F07017CA09917006F0706
+:10035000017C012093559D000700A7D9F598D36C27
+:100360006907047D6807027D010E64999358D600E1
+:10037000017D8E009355A005935DA006027802557D
+:10038000C86D0F7C004E087C6907037D0255097E0D
+:100390007199C86D067F890693500048017D5C996C
+:1003A000A0999A99C36A6907047D6807027D010EC6
+:1003B00087999358D600017D8E009355A005935DD3
+:1003C000A0060278C865045D0F7C004E087C6907B2
+:1003D000037DC865097E9499045D067F8906935064
+:1003E0000048017D7F99A09993559D000700FF6CFF
+:1003F000A7D9F5980000E354EB55004D017CF59822
+:10040000DD98E354EB55FF0A1102FF1A7F07027CC7
+:10041000A005B4999D008C05BA05A0051002BA0488
+:10042000AD0454040600E3C1DB57FB52C36AF35228
+:10043000056A8F00D500017D8D00A005EB5D780475
+:10044000037D79042B7D1E7C7904337CEE56000FEE
+:10045000FB556007027DC36DD599041DC36DC8624D
+:100460003B7E6006027D10021202096A357F12028D
+:10047000096A327F1202096A2F7F011F0320004898
+:10048000E77C099AFB55C76D150015001500057826
+:10049000C8620B6AC8620B6AC76D089AFB55C76DC4
+:1004A000150015000578C8620A6AC8620A6AC76D35
+:1004B000089AFB55C76D15000578C862096AC862BD
+:1004C000096AC76D097C286A077F0000EB55004D5B
+:1004D000057DFAC1DB57BF9977C254040AC2BA99A5
+:1004E000D9C1E3C1DB57F352056A8F00D500017D06
+:1004F0008D00A005FB567804037D7904297D1F7CBF
+:1005000079042E7CE35D700D1105ED55000F600739
+:10051000027D0652329A2652337E6005027D100219
+:100520001202096A2D7F1202096A2A7F1202096AE1
+:10053000277F011F03200048EA7CE3555D9A1500E0
+:1005400015001500047806520B6A26520B6A5C9A55
+:1005500015001500047806520A6A26520A6A5C9A47
+:10056000150004780652096A2652096A097C286A2D
+:10057000077F0000DB57004D057DFAC1DB571B9A52
+:1005800077C254040AC2189AE3C1DB57F352056AD2
+:10059000FB568E02941AC36AC8626902247D941EB7
+:1005A000C36ED36EC8624802C86A9426981EC36E92
+:1005B000D36EC8624C02C86A9826C36E981EC36E7A
+:1005C000C8629826C36E6002097CC8626E02247DF0
+:1005D000096A1E7F0125004D257D849A286A187FAF
+:1005E00004627AC2B89AE36E8F00D805017D8D004F
+:1005F000A005C8626E02107D096A0A7F0120F97C9D
+:10060000286A067F0000004D0D7DFAC1DB576E9A07
+:10061000070004620C6AB59A286AFA7F04627AC2FB
+:1006200058045404286AF47F0AC26B9AD9C1E3C102
+:10063000DB57F352056AFB568E02941A0252690286
+:100640001D7D941E06524802065A9426981E065294
+:100650004C02065A9826981E065260020A7C98267A
+:1006600006526E02237D096A1D7F0125004D247DFF
+:10067000D19A286A177F04627AC2029B8F00D8053C
+:10068000017D8D00A00506526E02107D096A0A7F69
+:100690000120F97C286A067F0000004D0D7DFAC11B
+:1006A000DB57C19A070004620C6AFF9A286AFA7F36
+:1006B00004627AC258045404286AF47F0AC2BE9ABB
+:1006C000016E0B612F7E0B622D7E0B632B7E0C0D5A
+:1006D0001704170417049D04081DCC05017C0C0D9C
+:1006E000D16A000F4207C86FDD6F1C7F8E009D002E
+:1006F00001680B67177ED56B04080278C86F120774
+:10070000117C0B670F7E04080278C86F12070A7C01
+:10071000DD6F087FD169010FC86FDD6F037F0101B5
+:0E0720000004129B0700FF680C680002129B89
 :00000001FF
-- 
1.7.9.5

